examples/responses/responseActivity1.jsonld

```diff
@@ -0,0 +1,20 @@
+{
+    "@context": "../../contexts/generic",
+    "@type": "reproschema:ResponseActivity",
+    "@id": "uuid:96496a96-50d6-4b87-a65e-468420e7c80d",
+    "prov:used": [
+        "https://raw.githubusercontent.com/ReproNim/reproschema/master/examples/activities/items/item1.jsonld",
```

Comment:
for this specific example, i think relative paths is fine as it will resolve to the full absolute path.